src/backtrace.cc

```diff
@@ -0,0 +1,5 @@
+#include "src/base/logging.cc"
```

Comment:
I included the .cc so it works with shared V8 builds (think Fedora, Debian, Ubuntu, etc.,or linking against a .so.)

I'll ifdef out smartos.  That user base is a rounding error compared to the aforementioned distros.